### DISCUSSION TITLE
Remove Downloaded Model

### DISCUSSION THREAD

**Discussion by EliasPereirah** [openai/whisper#762] [Q&A] [answered]
Hello, I'm finding Whisper amazing (Thanks OpenAi!).
I have a doubt, if anyone can enlighten me.
I downloaded the "model large" and my computer is not able to run it, when I run the command "whisper audio.mp3 --model large --language Portuguese" it returns "killed", I believe it is a ram memory limitation.
I will download the medium model, but to free up memory on the computer I would like to know how to remove the other model already downloaded.
I'm using Ubuntu on WSL.
Thanks.

**Comment by krylm** [comment, marked as answer]
Check directory
`~/.cache/whisper/`
You should have the large model there (large-v1.pt / large-v2.pt / large.pt file).

**Reply by EliasPereirah** [reply]
Thank you very much.
Solved.

**Comment by sanjibroy360** [comment]


**Reply by phineas-pta** [reply]
wrong repo, u r using https://github.com/ggerganov/whisper.cpp

**Reply by berupe** [reply]
What would be the cache line to check for the ggerganov version? 😅

**Reply by berupe** [reply]
fwiw I'm using terminal on Mac

**Comment by glangford** [comment]
> What would be the cache line to check for the ggerganov version? 😅

@berupe Check `~/.cache/whisper/`, but if you don't see it there you will need to ask on the ggerganov repo.
